submissions/ses3332002/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,415 @@
+            class="seat"
+          ></label>
+          <input type="checkbox" id="seat7_5" disabled /><label
+            for="seat7_5"
+            class="seat"
+          ></label>
+          <input type="checkbox" id="seat7_6" /><label
+            for="seat7_6"
+            class="seat"
+          ></label>
+          <input type="checkbox" id="seat7_7" disabled /><label
+            for="seat7_7"
+            class="seat"
+          ></label>
+          <input type="checkbox" id="seat7_8" /><label
+            for="seat7_8"
+            class="seat"
+          ></label>
+          <input type="checkbox" id="seat7_9" disabled /><label
+            for="seat7_9"
+            class="seat"
+          ></label>
+          <input type="checkbox" id="seat7_10" /><label
+            for="seat7_10"
+            class="seat"
+          ></label>
+          <input type="checkbox" id="seat7_11" disabled /><label
+            for="seat7_11"
+            class="seat"
+          ></label>
+          <input type="checkbox" id="seat7_12" /><label
+            for="seat7_12"
+            class="seat"
+          ></label>
+          <input type="checkbox" id="seat7_13" disabled /><label
+            for="seat7_13"
+            class="seat"
+          ></label>
+          <input type="checkbox" id="seat7_14" /><label
+            for="seat7_14"
+            class="seat"
+          ></label>
+        </div>
+        <div class="auditorium__row">
+          <input type="checkbox" id="seat8_1" data-type="vip" /><label
+            for="seat8_1"
+            class="seat"
+          ></label>
+          <input type="checkbox" id="seat8_2" data-type="vip" /><label
+            for="seat8_2"
+            class="seat"
+          ></label>
+          <input type="checkbox" id="seat8_3" data-type="vip" /><label
+            for="seat8_3"
+            class="seat"
+          ></label>
+          <input type="checkbox" id="seat8_4" data-type="vip" /><label
+            for="seat8_4"
+            class="seat"
+          ></label>
+          <input
+            type="checkbox"
+            id="seat8_5"
+            disabled
+            checked
+            data-type="vip"
+          /><label for="seat8_5" class="seat"></label>
+          <input type="checkbox" id="seat8_6" data-type="vip" /><label
+            for="seat8_6"
+            class="seat"
+          ></label>
+          <input type="checkbox" id="seat8_7" data-type="vip" /><label
+            for="seat8_7"
+            class="seat"
+          ></label>
+          <input type="checkbox" id="seat8_8" data-type="vip" /><label
+            for="seat8_8"
+            class="seat"
+          ></label>
+        </div>
+        <input type="submit" class="seats_submit" value="Заказать" />
+      </form>
+      <aside class="auditorium__description">
+        <div class="description"></div>
+        <div class="description description-vip"></div>
+        <div class="description__text">- свободно</div>
```

Comment:
Logically better to group these blocks.
```suggestion
        <div class="description">
            <p class="description__text">
                <span class="description__seat"></span>
                <span class="description__seat description__seat-vip"></span>
             - свободно </p>
        </div>
```

Reply:
I've remade it